<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map editor</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Edit</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="structures_tab">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="mainbuilds">
          <attribute name="title">
           <string>Mainbuilds</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QListWidget" name="mb_struct_list">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::NoDragDrop</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="bg_decorations">
          <attribute name="title">
           <string>BG</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QListWidget" name="bgd_struct_list">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="dynamic_structures">
          <attribute name="title">
           <string>Dynamic</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QListWidget" name="ds_struct_list">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="spikes">
          <attribute name="title">
           <string>Spikes</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QListWidget" name="sk_struct_list">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="decorations">
          <attribute name="title">
           <string>Decorations</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QListWidget" name="dec_struct_list">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="fg_decorations">
          <attribute name="title">
           <string>FG</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QListWidget" name="fgd_struct_list">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="special_objects">
          <attribute name="title">
           <string>Special objects</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QListWidget" name="so_struct_list">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="objects_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>Objects</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QListWidget" name="obj_list">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphics_view">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Infos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>248</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>242</width>
             <height>248</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QGroupBox" name="data_box">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>200</height>
               </size>
              </property>
              <property name="title">
               <string>Data</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="4" column="1">
                <widget class="QLabel" name="bbox_label">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Bounding box:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="resource_label">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Resource:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="y_spinbox">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>-20000</number>
                 </property>
                 <property name="maximum">
                  <number>20000</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="layer_spinbox">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>-10</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Position x:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Layer:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Position y:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="name_lineedit">
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="x_spinbox">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>-20000</number>
                 </property>
                 <property name="maximum">
                  <number>20000</number>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Object name:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="preview_box">
          <property name="title">
           <string>Preview</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QGraphicsView" name="preview_graphics_view">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="triggers_tab">
       <attribute name="title">
        <string>Triggers</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="cp_tab">
       <attribute name="title">
        <string>CP</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>tabs</zorder>
   <zorder>graphics_view</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_new_struct"/>
    <addaction name="action_remove_current_struct"/>
    <addaction name="action_edit_current_struct"/>
    <addaction name="separator"/>
    <addaction name="action_set_bg"/>
   </widget>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_new"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="action_delete_selected_structure"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="action_zoom_plus"/>
     <addaction name="action_zoom_minus"/>
    </widget>
    <addaction name="menuZoom"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="menuSelection"/>
   <addaction name="menuDisplay"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_load_struct">
   <property name="text">
    <string>Load from disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_new_struct">
   <property name="text">
    <string>New structure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_set_bg">
   <property name="text">
    <string>Set background</string>
   </property>
  </action>
  <action name="action_set_href">
   <property name="text">
    <string>Set height reference</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_delete_selected_structure">
   <property name="text">
    <string>Delete selected object</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_zoom_plus">
   <property name="text">
    <string>Zoom +</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_zoom_minus">
   <property name="text">
    <string>Zoom -</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_remove_current_struct">
   <property name="text">
    <string>Remove structure</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="action_edit_current_struct">
   <property name="text">
    <string>Edit structure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
